composables/transaction/transactionConsume.ts

```diff
@@ -0,0 +1,24 @@
+import { Interaction, createInteraction } from '@kodadot1/minimark'
+
+import { bsxParamResolver, getApiCall } from '@/utils/gallery/abstractCalls'
+import type { ActionConsume } from './types'
+
+export function exectConsumeTx(item: ActionConsume, api, executeTransaction) {
+  if (item.urlPrefix === 'rmrk') {
+    executeTransaction({
+      cb: api.tx.system.remark,
+      arg: [createInteraction(Interaction.CONSUME, '1.0.0', item.nftId, '🔥')],
```

Comment:
I'm not sure if the '🔥' is properly here TBH

Reply:
not needed, it's only for the style 💅

Reply:
sooo, let's update it ?

Reply:
wut? you want me to remove this fancy emoji?

Reply:
Yes, I guess we could display the fancy emoji from the code in frontend. I'm not sure if we should write the emoji into the metadata of nft. WDYT @vikiival

Reply:
Oops didn't knew about this one https://github.com/kodadot/packages/blob/544b0f29ef9f3ab7257ba82c60c21ae70647b4b1/minimark/src/rmrk/create.ts#L8 
I'll remove it

Reply:
@Jarsen136 is correct `createInteraction` is internal minimark package.

https://github.com/rmrk-team/rmrk-spec/blob/master/standards/rmrk1.0.0/interactions/consume.md#standard

however our rubick indexer can process it without any problem

https://github.com/kodadot/rubick/blob/14e0827b5148cd328b16a7e85d5dec9030403cc3/src/mappings/index.ts#L249

What am I afraid of that RMRK consolidator can b*tch about extra field in the remark string

So for my mental sanity:
- please remove it from the `createInteraction`
- happy to have as many 🔥 emoji on frontend as possible